packages/fxa-auth-server/lib/metrics/events.js

```diff
@@ -224,7 +224,7 @@ module.exports = (log, config) => {
     return log.flowEvent(
       Object.assign({}, data, {
         event: `route.performance.${path}`,
-        flow_time: Date.now() - request.info.received,
+        flow_time: request.info.responded - request.info.received,
```

Comment:
In this case, I think the use of `request.info.responded` is fine, since this is only invoked if the status is 2xx (line 166). Might be worth documenting that this function expects success responses, in case it's reused to log error info in the future?